restructuredtext_lint/lint.py

```diff
@@ -80,9 +75,24 @@ def error_collector(data):
     return errors
 
 
-def lint_file(filepath, encoding=None):
+def lint(content, filepath=None, ignore_sphinx=True):
```

Comment:
Sure, how about enums for [python, cpp, c, javascript, std] since thats all the ones that are possible (afaik); can use https://pypi.python.org/pypi/enum34 to help here...